src/main/java/com/openshift/restclient/utils/EnvironmentVariableUtils.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift.restclient.utils;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.openshift.restclient.model.IEnvironmentVariable;
+
+
+/**
+ * @author Andre Dietisheim
+ */
+public class EnvironmentVariableUtils {
+	
+	private EnvironmentVariableUtils(){
+	}
+	
+	public static  Map<String, String> toMapOfStrings(Collection<IEnvironmentVariable> envVars){
```

Comment:
We may already do something similiar for DC, RC, Pods, Deployment.  Care to investigate and possibly refactor?  Maybe follow on issue if you find something?